File: PHP/Program/SivitasAkademik.php
<?php

/* Saya Hakasa Putri mengerjakan Latihan 2 DPBO 2023 C2 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */


include "Human.php"; //import class Human

class SivitasAkademik extends Human{ //class Sivitas Akademik turunan dari class Human

    //atribut private class Sivitas Akademik

    private $asal_univ;
    private $email_edu;

    public function __construct(){ //constructor

    }

    //setter getter

	public function getasal_univ() {
		return $this->asal_univ;
	}

	public function setasal_univ($asal_univ) {
		$this->asal_univ = $asal_univ;
	}

	public function getemail_edu() {
		return $this->email_edu;
	}

	public function setemail_edu($email_edu){
		$this->email_edu = $email_edu;
	}

    //Tampilkan atribut class sivitas akademik
    public function getInfoSivitasAkademik(){

        echo "Asal Universitas  : ". $this->getasal_univ()."<br/>";
        echo "email_edu         : ". $this->getemail_edu()."<br/>";
        
    } 

	public function __destruct(){ //desctructor
		
	}
}

File: Python/Program/SivitasAkademik.py
# Saya Hakasa Putri mengerjakan Latihan 2 DPBO 2023 C2 dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
# kecurangan seperti yang telah dispesifikasikan. Aamiin 

from Human import Human

class SivitasAkademik(Human): # class Sivitas Akademik turunan dari class Human

    def __init__(self): #Constructor

        # atribut
        self.asal_univ = "";
        self.email_edu = "";


    # setter getter

    def getasal_univ(self):
        return self.asal_univ
    
    def setasal_univ(self, asal_univ):
        self.asal_univ = asal_univ
    
    def getemail_edu(self):
        return self.email_edu
        
    def setemail_edu(self, email_edu):
        self.email_edu = email_edu
    

    # untuk print atribut kelas Sivitas Akademik
    def getInfoSivitasAkademik(self):
        print("Asal Universitas  : " + self.getasal_univ())
        print("Email Edu         : " + self.getemail_edu())
